### ISSUE TITLE
[Bug]: Terraform extension parameter does not work

### ISSUE THREAD

**Issue by stemons** [awslabs/aws-deployment-framework#602]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Terraform extension parameter is not working as expected

### Expected Behavior

When the terraform extension parameter is set to true the Dynamo DB table (state file lock) should be created in the deployment account in the ADF main region and all the other regions enabled

### Current Behavior

When the terraform extension parameter is set to true the Dynamo DB table (state file lock) has not been created in the deployment account in the ADF main region and all the other regions enabled

### ADF Version

3.2.0

**Comment by LucaCominoli21** [comment]
Same problem, i think it's because the parameter `/adf/extensions/terraform/enabled` has value `True` but CloudFormation condition wants `true` as value.

I fixed modifying, in bootstrap repo at the path "adf-bootstrap/deployment/regional.yml", the condition "ADFTerraformExtensionEnabled"

from:

```
Conditions:
  ADFTerraformExtensionEnabled:
    Fn::Equals:
      - !Ref ADFTerraformExtension
      - true
```

to:

```
Conditions:
  ADFTerraformExtensionEnabled:
    Fn::Equals:
      - !Ref ADFTerraformExtension
      - "True"
```

In this way the condition is evaluated correctly.

**Comment by sbkok** [comment]
Thank you for your patience. I am pleased to inform you that this issue has been [resolved in our latest release v4.0.0](https://github.com/awslabs/aws-deployment-framework/releases/tag/v4.0.0) just now.
I'm hereby closing this issue. Please open a new issue if you are experiencing any issues with the latest release.
